site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -105,9 +91,9 @@ In addition, the port `9011` is mapped between the container and your host machi
 
 You may add other services to the Docker Compose files if needed.
 
-For example, in development environments it can be helpful to run https://mailcatcher.me/[MailCatcher] for SMTP.
+For example, in development environments it can be helpful to run https://mailcatcher.me/[MailCatcher] which provides a local SMTP server. This allows FusionAuth to send emails for account verification or a password reset.
 
-[source,yaml,title=`docker-compose.override.yml with mailcatcher]
+[source,yaml,title=`docker-compose.yml with mailcatcher]
 ----
 version: '3'
 
```

Comment:
should we add a bit about the volumes?

```
networks:
  db_net:
    driver: bridge
  search_net:
    driver: bridge
  mailcatcher_net:
    driver: bridge
```

And rename to mailcatcher_net to keep with the update naming convention?:

Reply:
good catch. Updated.